homeassistant/components/mill/climate.py

```diff
@@ -213,7 +215,7 @@ class LocalMillHeater(CoordinatorEntity[MillDataUpdateCoordinator], ClimateEntit
     def __init__(self, coordinator: MillDataUpdateCoordinator) -> None:
         """Initialize the thermostat."""
         super().__init__(coordinator)
-        self._attr_name = coordinator.mill_data_connection.name
+        self._attr_name = None
```

Comment:
Same change/suggestion as above. Unfortunately, GitHub doesn't allow me to suggest on this one like the above.